<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>898</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>End of Day: Staff</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>123</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="dayInfoGrid">
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="staffEdit"/>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="storeLabel">
       <property name="text">
        <string>Store</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="staffLabel">
       <property name="text">
        <string>Staff</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="storeCombo">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plaza</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Village</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dateDisplay">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="dateSelectButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="eftGrid">
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="eftActualLabel">
       <property name="text">
        <string>Actual</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="eftRegisterLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Register&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="eftRegisterEdit"/>
     </item>
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="eftLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Eftpos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="eftEnterButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Enter Values</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="epayGrid">
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="epayLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Epay&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="epayRegisterLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Register&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="epayRegisterEdit"/>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="epayActualLabel">
       <property name="text">
        <string>Actual</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="epayActualEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="scratchiesGrid">
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="scratchiesLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Scratchies&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="scratchiesRegisterEdit"/>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="scratchiesActualLabel">
       <property name="text">
        <string>Count</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="scratchiesRegisterLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Register&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="scratchiesActualEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="scratchiesPayGrid">
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="scratchiesPayActualLabel">
       <property name="text">
        <string>Lotto</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="scratchiesPayRegisterLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Register&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="scratchiesPayRegisterEdit"/>
     </item>
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="scratchiesPayLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Scratchie Payouts&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="scratchiesPayEnterButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Enter Values</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="lottoPayGrid">
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lottoPayActualLabel">
       <property name="text">
        <string>Lotto</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lottoPayLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Lotto Payouts&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lottoPayRegisterLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Register&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lottoPayRegisterEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="lottoPayEnterButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Enter Values</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="lottoGrid">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lottoRegisterEdit"/>
     </item>
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lottoLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Lotto&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lottoActualLabel">
       <property name="text">
        <string>Lotto</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lottoRegisterLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Register&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="lottoEnterButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Enter Values</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>261</width>
      <height>370</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="cashGrid">
     <item row="8" column="1">
      <widget class="QLineEdit" name="cashCoinsEdit"/>
     </item>
     <item row="7" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cash5Label">
       <property name="text">
        <string>$5</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="cash5Edit"/>
     </item>
     <item row="4" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cash50Label">
       <property name="text">
        <string>$50</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cash100Label">
       <property name="text">
        <string>$100</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cashregisterLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Register&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cash20Label">
       <property name="text">
        <string>$20</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cash10Label">
       <property name="text">
        <string>$10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="cash100Edit"/>
     </item>
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="cashLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Cash&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="cash50Edit"/>
     </item>
     <item row="8" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cashCoinsLabel">
       <property name="text">
        <string>Coins</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="cashAsideEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="cash10Edit"/>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cashAsideLabel">
       <property name="text">
        <string>Put Aside</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="cashRegisterEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="cash20Edit"/>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="cashPayoutsLabel">
       <property name="text">
        <string>Payouts Used</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cashPayoutsEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="lineV1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>20</width>
      <height>631</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="lineH3">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>120</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="ineH4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineH5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineH6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineH2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineH1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineH7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>790</y>
      <width>541</width>
      <height>36</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Generate EOD File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>840</y>
      <width>541</width>
      <height>36</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Print EOD</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>657</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="notesGrid">
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="notesLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Notes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="notesEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="lineH7_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
